### ISSUE TITLE
Move top position to bottom on long list

### ISSUE THREAD

**Issue by XbIK** [hanshengchiu/reorderables#17]
Wonderful package! Get one issue
Use **ReorderableSliverChildBuilderDelegate**, if the list is big and  I move the very first position (index [0]) to the bottom , so the FlexibleSpaceBar is collapsed and top of the list is not visible, after that the new index[0] is not draw on top.  I have to reopen the page and it appears.

**Comment by hanshengchiu** [comment]
did it happen when you were using ReorderableSliverChildListDelegate?

**Comment by XbIK** [comment]
Yes, with Delegate, and only when the list is big enough, so the top of it moves and hide FlexibleSpaceBar

**Comment by FelixYew** [comment]
Thanks for the cool package! I also facing the same problem the first item will disappear when dragging to the bottom of a very long list. Any workaround?

**Comment by kevincherryholme** [comment]
Any idea if there is a workaround for this? Really need this fixed.
